project-react/src/pages/Tablee.js

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import "./Tablee.css";
+const Tablee = () => {
+  const [posts, setPosts] = useState([]);
+  const apiEndPoint = "https://api.github.com/users/ariv797/repos";
+  useEffect(() => {
+    const getPosts = async () => {
+      const { data: res } = await axios.get(apiEndPoint);
+      setPosts(res);
+    };
+    getPosts();
+  }, []);
+
+
+  if (posts.length === 0) return <h2> There are no Student in the Database </h2>;
+  return (
+    <>
+      <div className="container">
+        <h2> There are {posts.length} Student in the Database </h2>
+        <table className="table">
+          <thead>
+            <tr>
+              
+              <th>Id</th>
+              <th>Full_Name</th>
+              <th>Type</th>
+              <th>Size</th>
+              <th>Language</th>
+              <th>Stargazers_Count</th>
+              <th>Watchers_Count</th>
+              <th>Visibility</th>
+              <th>Default_Branch</th>
+            </tr>
+          </thead>
+          <tbody>
+            {posts.map((post) => (

```

Comment:
maybe u should add key to tr 
console may show to u that there is bug which says each element in map should have a unique key 
could u check if this error shows in console ?

Reply:
Yes correct. 
An error was occurring. When I added the key, the problem was solved.